components/dashboard/src/components/RepositoryFinder.tsx

```diff
@@ -157,3 +189,25 @@ function saveSearchData(searchData: string[]): void {
         console.warn("Could not save search data into local storage", error);
     }
 }
+
+function filterRepos(searchString: string, suggestedRepos: SuggestedRepository[]) {
+    let results = suggestedRepos;
+    const normalizedSearchString = searchString.trim().toLowerCase();
+
+    if (normalizedSearchString.length > 1) {
+        results = suggestedRepos.filter((r) => {
+            return `${r.url}${r.projectName || ""}`.toLowerCase().includes(normalizedSearchString);
```

Comment:
**issue**: When I add a project and navigate to workspaces and then /new the project is not listed there unless I refresh or hard reload. Is this known?

Reply:
It should update/refresh that list with the latest to include any new projects or renamed projects (might show stale data for a second). I can make it better and make sure we invalidate some cached data when new projects are created or renamed though.